tests/components/geofency/test_init.py

```diff
@@ -316,5 +318,5 @@ async def test_load_unload_entry(hass, geofency_client, webhook_id):
     assert state_1 is not state_2
 
     assert STATE_HOME == state_2.state
-    assert state_2.attributes["latitude"] == HOME_LATITUDE
-    assert state_2.attributes["longitude"] == HOME_LONGITUDE
+    assert state_2.attributes[ATTR_LATITUDE] == HOME_LATITUDE
+    assert state_2.attributes[ATTR_LONGITUDE] == HOME_LONGITUDE
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)